docs/pages/reference/backends.mdx

```diff
@@ -116,6 +116,11 @@ To configure Teleport for using etcd as a storage backend:
     etcd](https://github.com/etcd-io/etcd/tree/master/hack/tls-setup) if you
     don't already have a TLS setup.
 - Configure all Teleport Auth servers to use etcd in the "storage" section of the config file as shown below.
+- Each Teleport Auth server will have individual audit logs within the `log` directory under the data
+  directory (by default `/var/lib/teleport/log`). Using a SIEM to consolidate audit logs is recommended.
```

Comment:
If we're going to recommend using a SIEM to consolidate audit logs, we might want to link to the section of the docs re: setting up the Event Handler for different services.